crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -235,24 +235,27 @@ impl<'a> LinkStage<'a> {
 
       // TODO: should have a better place to put this
       if is_entry && matches!(self.options.format, OutputFormat::Cjs) {
-        importer.star_exports.iter().for_each(|rec_idx| {
-          let rec = &importer.import_records[*rec_idx];
-          match &self.module_table.modules[rec.resolved_module] {
-            Module::Normal(_) => {}
-            Module::External(ext) => {
-              self.metas[importer.idx]
-                .require_bindings_for_star_exports
-                .entry(rec.resolved_module)
-                .or_insert_with(|| {
-                  // Created `SymbolRef` is only join the de-conflict process to avoid conflict with other symbols.
-                  self.symbols.create_facade_root_symbol_ref(
-                    importer.idx,
-                    legitimize_identifier_name(&ext.name).into_owned().into(),
-                  )
-                });
+        importer
+          .import_records
```

Comment:
The same as https://github.com/rolldown/rolldown/pull/2497/files#r1814691768.

Reply:
This one is still slower though it have o(1) check `has_star_export`. Anyway, probably should investigate it when we have profiling data.

Reply:
Yeah, but it needs to meet `is_entry` so only a few module will be executed.